packages/cpt-ui/app/searchforaprescription/page.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import React from "react";
+import { Col, Container, Hero, Row } from "nhsuk-react-components";
+//import "../../assets/styles/prescriptionSearch.scss"
+import "../../assets/styles/searchforaprescription.scss"
```

Comment:
This should use a path, 
```
import "@/assets/styles/searchforaprescription.scss"
```

You may need to add this to the `tsconfig.json` file:
```
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/context/*": ["context/*"],
      "@/constants/*": ["constants/*"],
      "@/assets/*": ["assets/*"],
      "@/components/*": ["components/*"]
    },
    ...
  }
}
```

Similar for components and constants.

Reply:
yep, good spot